tests/requirements.txt

```diff
@@ -1,2 +1,4 @@
 docker-compose==1.28.5
+mock==4.0.3
```

Comment:
We really should just change the imports to use `unittest.mock` since we don't need to support Python 2.7 anymore, but this is fine for now.